src/Symfony/Component/Cache/Traits/ApcuTrait.php

```diff
@@ -52,11 +52,14 @@ private function init($namespace, $defaultLifetime, $version)
     protected function doFetch(array $ids)
     {
         try {
+            $values = array();
             foreach (apcu_fetch($ids, $ok) ?: array() as $k => $v) {
                 if (null !== $v || $ok) {
-                    yield $k => $v;
+                    $values[$k] = $v;
                 }
             }
+
+            return $values;
```

Comment:
why not using an iterator anymore here ?

Reply:
because it's faster, and the data is already in memory so the iterator has not benefit